### PR TITLE
Updating assurance project publications repo

### PR DESCRIPTION
**Pull request by Carampearson**
Uploaded 2 reports and assurance projects jan - apr 2021

### CODE REVIEW

**Comment by dguido** [comment]
Are these all in the right category? E.g., Flare Network is not a technology project.

**Comment by Carampearson** [comment]
They SHOULD be, Flare was included in technology because the smart contract review is separate...  was scoped to provide a security assessment of the Flare codebase, including its use of the Avalanche codebases (avalanchego and coreth).

File: README.md [hunk 1]
```diff
@@ -209,6 +209,11 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Review Date | Level of Effort | Deliverables | Announcement |
 | --- | --- | --- | --- | --- |
+| [VGS Proxy](https://www.verygoodsecurity.com/) | Apr 2021 | 4 person-weeks | |
+| [Flare Network](https://flare.xyz/) | Mar 2021 | 8 person-weeks | |
```
Comment (mgcolburn) [inline README.md:213]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -314,6 +333,13 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Review Date | Level of Effort | Announcement |
 | --- | --- | --- | --- |
+| [Arbitrum V2](https://offchainlabs.com/) | Feb 2021 | 8 person-weeks | |
```
Comment (mgcolburn) [inline README.md:336]:
```suggestion
| [Flare Network](https://flare.xyz/) | Mar 2021 | 8 person-weeks | |
| [Arbitrum V2](https://offchainlabs.com/) | Feb 2021 | 8 person-weeks | |
```

**Review by Carampearson — COMMENTED** [review]
Michael moved Flare, looks good to me

**Review by withzombies — APPROVED** [review]

**Pull request merged**